src/helpers/errorHandlers.ts

```diff
@@ -53,6 +53,10 @@ export function generateErrorMap(intl: IntlShape): IErrorMap {
             message: intl.formatMessage({ id: "visualization.ErrorMessageGeneric" }),
             description: intl.formatMessage({ id: "visualization.ErrorDescriptionGeneric" }),
         },
+        [ErrorStates.GEO_MAPBOX_TOKEN_MISSING]: {
+            message: intl.formatMessage({ id: "visualization.ErrorMessageMissingMapboxToken" }),
```

Comment:
it should be replaced to `visualization.ErrorMessageGeneric`

Reply:
fixed